portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="topnav">
+            <div>
+                <a class="mylogo" href="#mylogo">My Logo</a>
+            </div>
+            <ul class="nav__links">
+                <!-- <li class="line"><a href="#portfolio">Portfolio</a></li>
+                <li class="line"><a href="#about">About</a></li>
+                <li class="line"><a href="#contact">Contact</a></li> -->
```

Comment:
- [ ] Pease delete commented code to keep the html document clean 😉

Reply:
Addressed 🧐